src/main/scala/cc/factorie/app/nlp/ner/StackedChainNer.scala

```diff
@@ -585,55 +585,10 @@ class StackedChainNer[L<:NerTag](labelDomain: CategoricalDomain[String],
   }
 }
 
-object StackedChainNer {
-  def initLexicons() : Unit = synchronized {
-    lexicon.iesl.Month.toString()
-    lexicon.iesl.Day.toString()
-
-    lexicon.iesl.PersonFirst.toString()
-    lexicon.iesl.PersonFirstHigh.toString()
-    lexicon.iesl.PersonFirstHighest.toString()
-    lexicon.iesl.PersonFirstMedium.toString()
-
-    lexicon.iesl.PersonLast.toString()
-    lexicon.iesl.PersonLastHigh.toString()
-    lexicon.iesl.PersonLastHighest.toString()
-    lexicon.iesl.PersonLastMedium.toString()
-
-    lexicon.iesl.PersonHonorific.toString()
-
-    lexicon.iesl.Company.toString()
-    lexicon.iesl.JobTitle.toString()
-    lexicon.iesl.OrgSuffix.toString()
-
-    lexicon.iesl.Country.toString()
-    lexicon.iesl.City.toString()
-    lexicon.iesl.PlaceSuffix.toString()
-    lexicon.iesl.USState.toString()
-    lexicon.iesl.Continents.toString()
-
-    lexicon.wikipedia.Person.toString()
-    lexicon.wikipedia.Event.toString()
-    lexicon.wikipedia.Location.toString()
-    lexicon.wikipedia.Organization.toString()
-    lexicon.wikipedia.ManMadeThing.toString()
-    lexicon.iesl.Demonym.toString()
-
-    lexicon.wikipedia.Book.toString()
-    lexicon.wikipedia.Business.toString()
-    lexicon.wikipedia.Film.toString()
-
-    lexicon.wikipedia.LocationAndRedirect.toString()
-    lexicon.wikipedia.PersonAndRedirect.toString()
-    lexicon.wikipedia.OrganizationAndRedirect.toString()
-  }
-}
-
```

Comment:
@JackSullivan I applaud the move to a more explicit style for loading the lexicons from input streams (Speaking of which, perhaps the examples in the scaladoc should buffer the input stream (with a bigish buffer) before making a DataInputStream so that you're not getting a zillion direct reads from the IO subsystem if people follow the examples?)

I have a strong anti-magic mindset (I am old and disillusioned, you see, so the wonder of magic is lost on me), so having the option to do something explicit like:

val foo = new Lexicon(someInputStream)
val bar = new Lexicon(someOtherInputStream)
val model = new SomeCleverModel(<stuff>, foo, bar)

would make me more comfortable about doing things in weird/multithreaded/weird and multithreaded environments. And I definitely don't want to have multiple copies of the lexicons around, since they're huge (that's partly my fault, though, so I won't complain too loudly.)

Lazy eval still worries me a bit, because I'm worried the Scala guys haven't pushed things very hard and because you pay a little bit every time you use the lexicon (A short play: "Check a word? Sure thing! Better check if I have my data first!. Hope this is thread safe! Whoops, I guess I have data, so I can go check that word now!")  Basically, if I go to the trouble of writing the code to load a lexicon, I want that code to cause a lexicon to be loaded and I expect when I return from the call that I can ask about words without further delay.